README.md

```diff
@@ -40,60 +45,6 @@ npm install
-8. Call `initialize` method at Home Bridge Proxy address with 4 parameters:
-- address of Home Bridge **Validators** address Proxy ( step# 1) `0x01`
-- Daily Limit in wei: Example `1000000000000000000` == 1 ether
-- Maximum Per Transaction Limit in wei: Example `100000000000000000` == 0.1 ether. Should be less than Daily Limit
-- Minimum Per Transaction Limit in wei: Example `10000000000000000` == 0.01 ether. Should be less than Daily Limit and less than MaxPerTx
-=====
-## Foreign Deployment on Kovan
-=====
-
-9. Deploy [POA20 contract](https://github.com/poanetwork/poa-parity-bridge-contracts/blob/upgradable/flats/POA20_flat.sol#L448) Example(`0x05`)
-10. Deploy [EternalStorageProxy contract](https://github.com/poanetwork/poa-parity-bridge-contracts/blob/upgradable/flats/EternalStorageProxy_flat.sol#L218)
-that will be used as Foreign Bridge Validators Proxy contract. Example `0x06`
-11. Deploy [BridgeValidators contract](https://github.com/poanetwork/poa-parity-bridge-contracts/blob/upgradable/flats/BridgeValidators_flat.sol#L197) Example `0x07`
-12. Call `upgradeTo` of EternalStorageProxy(`0x06`) that is used as  Foreign Bridge Validators Proxy(step#10) with 2 parameters:
-- 0 - version of implementation contract
-- address of implementation which is the address of step#11 Foreign BridgeValidators deployed contract. (`0x07`)
-13. Call `initialize` method at Foreign Bridge Validators Proxy(`0x06`) with 2 parameters:
-- REQUIRED_NUMBER_OF_VALIDATORS. Example: `1`
-- Array of validators. Example `["0x0039F22efB07A647557C7C5d17854CFD6D489eF3", "0xf052d236b8076879d86c9e4c116a068a0d420c55"]`
-14. Deploy [EternalStorageProxy contract](https://github.com/poanetwork/poa-parity-bridge-contracts/blob/upgradable/flats/EternalStorageProxy_flat.sol#L218)
-that will be used as Foreign Bridge Proxy contract. Example `0x08`
-15. Deploy [Foreign Bridge contract](https://github.com/poanetwork/poa-parity-bridge-contracts/blob/upgradable/flats/ForeignBridge_flat.sol#L345) Example `0x09`
-16. Call `upgradeTo` of EternalStorageProxy(`0x08`) that is used as  Foreign Bridge Proxy(step#14) with 2 parameters:
-- 0 - version of implementation contract
-- address of implementation which is the address of step#15 Foreign Bridge deployed contract. (`0x09`)
-17. Call `initialize` method at Foreign Bridge Proxy(`0x08`) address with 5 parameters:
-- address of Foreign Bridge **Validators** address Proxy ( step# 10) `0x06`
-- address of POA20 token contract. Step#9 `0x05`
-- Daily Limit in wei: Example `1000000000000000000` == 1 ether
-- Maximum Per Transaction Limit in wei: Example `100000000000000000` == 0.1 ether. Should be less than Daily Limit
-- Minimum Per Transaction Limit in wei: Example `10000000000000000` == 0.01 ether. Should be less than Daily Limit and less than MaxPerTx
-18. Call `transferOwnership` of POA20 contract(0x05) with 1 parameter:
-- address of Foreign Bridge Proxy (`0x08`) from step#14
-
-
 # To Flatten
```

Comment:
Can we add a #Testing heading with any information about testing here - ie the testing command, any additional flags, etc.